styles/overlay.less

```diff
@@ -45,7 +45,7 @@ atom-panel.modal {
     li {
       padding: @component-padding;
       border-bottom: 1px solid fade(@overlay-border-color, 5%);
-      font-weight:500;
+      font-weight:600;
```

Comment:
San Francisco treats 500 like 400. Luckily 600 is the same weight as Helvetica Neue 500.